tests/integration/components/gleam-logo-test.js

```
@@ -0,0 +1,137 @@
+
+module('Integration | Component | gleam-logo', function (hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function () {
+    this.originalRive = window.Rive;
+    window.Rive = class MockRiveConstructor {
+      constructor(options) {
+        const instance = new MockRive(options);
+        this._onLoadCallback = options.onLoad;
+
+        return instance;
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)